xla/service/cpu/onednn_ops_rewriter.cc

```diff
@@ -84,6 +86,9 @@ auto MaxReduce(HloInstruction** instr) {
       m::Reduce(instr).WithPredicate(is_valid_reduce_max).WithOneUse());
 }
 
+// Match the softmax patter with divide instruction as root node.
```

Comment:
Use active voice (i.e. Match -> Matches) for function comments. See [style guide](https://google.github.io/styleguide/cppguide.html#Function_Comments).

Also: patter -> pattern.

Reply:
Done